core/src/main/kotlin/team/backend/exception/LayeredExceptionBundle.kt

```diff
@@ -0,0 +1,19 @@
+package team.backend.exception
+
+import team.backend.response.ResponseCode
+
+open class CommonException(
```

Comment:
@mangchhe This idea not bad. But I have recently been questioning whether this is the best way. 

The biggest reason is that when __monitored by Datadog, it can be filtered by ErrorType in APM.__

__Filtering:__

```
Search for: env:real error.type:org.dope.backend.server.api.base.exception.OutOfStockException
```

Therefore, I thought it would be more helpful to subdivide custom exceptions.

So, I think it would be better to inherit and use one BusinessException or CustomException, an abstract class that inherits RumtimeException.

Reply:
@BAEKJungHo 
I agree with your opinion. 
I was also attempting to subdivide custom exceptions, such as DomainException, which would inherit from CommonException. 
Are you suggesting that we should only have one abstract class that inherits from RuntimeException, without using layered exceptions?

Reply:
@BAEKJungHo 
There is a custom message within the responseCode.
What purpose do you mean custom message?

Reply:
@mangchhe I think ResponseCode message is default message.

```
// NotFoundResourceException has default value (responseCode)
throw NotFoundResourceException("customMessage")
```